pkgs/development/compilers/llvm/rocm/default.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, lib, buildPackages, fetchFromGitHub, callPackage, wrapCCWith, overrideCC }:
 
 let
-  version = "5.3.1";
+  version = "5.3.3";
```

Comment:
version changed, but hash stayed the same?

Reply:
Yes, this happens quite often with ROCm. All projects get a new tag for a minor version, but only some of them have actual changes.